### ISSUE TITLE
Improve the space deletion confirmation dialog

### ISSUE THREAD

**Issue by xcoulon** [openshiftio/openshift.io#2107]
The current message is as follow:

![screenshot 2018-01-30 10 42 27](https://user-images.githubusercontent.com/178502/35559165-4919acac-05aa-11e8-89b5-1a282de98eab.png)

IMHO, the message should at least contain the name of the space, so the user knows which space is about to be deleted. Or something even less error-prone, where the user needs to input the name of the space in order to confirm the deletion. Since the space contains a lot of elements (work items, codebase, etc.), it could be worth ensuring that the user did not delete the wrong space (or was too fast in cliking on the `Remove` button).

**Comment by aslakknutsen** [comment]
@xcoulon Where did you click this?

I believe it should follow similar flow as Reset Env, with a full list of what is being deleted and a confirmation box etc; https://redhat.invisionapp.com/share/2BD1C1GQY#/screens/249789313

**Comment by joshuawilson** [comment]
I think this is getting removed with the newer UX designs.
@catrobson can you confirm?

**Comment by catrobson** [comment]
Looks like we didn't resolve this in the design or dev of the spaces listing, but we have set out a standard for delete modals that I'd like to follow here:  https://redhat.invisionapp.com/share/YSDL0KRKW#/273202263_Deployments_Delete.   Does deleting a space actually work though?  I think there are a variety of problems to solve about what, other than the space construct, we delete (pipelines, codebases, etc).  Not sure if we should fix the modal or remove the option to delete entirely for now.

**Comment by joshuawilson** [comment]
This modal is on the View All spaces page. Are we keeping that page?

**Comment by catrobson** [comment]
Same modal from both My Spaces and View All Spaces.  Expecting to remove View All Spaces page when we can add a search to the My Spaces page.

**Comment by joshuawilson** [comment]
ah, ok
I'll leave this to you for design then.

**Comment by aslakknutsen** [comment]
@catrobson so this is out? https://redhat.invisionapp.com/share/2BD1C1GQY#/screens/249789313

**Comment by catrobson** [comment]
@aslakknutsen.  Correct, sorry, lost track of the design coming back from vacation!  That was my open question to Joshua (and you) though - are we going to do the backend work to support that full space delete.  Right now its hard to give any description of what it does since it just creates orphaned objects to my understanding.

**Comment by joshuawilson** [comment]
yes, that is work have planned for

**Comment by qodfathr** [comment]
@catrobson  I think all we really need to answer for right now is "of the two designs for the Space deletion confirmation dialog, which one is the right one?"

**Comment by catrobson** [comment]
This is the proposed design: https://redhat.invisionapp.com/share/2BD1C1GQY#/screens/249789131

**Comment by qodfathr** [comment]
PM approves design.
@joshuawilson Platform?

**Comment by joshuawilson** [comment]
Looks good to me. Matches the flow for reset env.

**Comment by aslakknutsen** [comment]
@qodfathr @joshuawilson I wouldn't sign on for the 'complete' space delete for H, but design wise agree it's a good path. :)

**Comment by joshuawilson** [comment]
I was only agreeing with the design, not committing to delivering it in H train.

**Comment by qodfathr** [comment]
Yes, I was only asking about the design itself, so that this may now move to the Approved Backlog, not the Train Backlog.
